### ISSUE TITLE
project status: abandoned or to be resurrected

### ISSUE THREAD

**Issue by ssbarnea** [kilianc/rtail#128]
I do find rtail as an useful project with great potential, probably being one of the few full open-source solutions in that area. Sadly looking at commits, issues, pull requests and forks I realise that is likely that is abandoned.

If there is no hidden agenda for killing that project, I would really love to see it revived and moved under its own github organization where there would be at least 3 people able to manage it. That's only to avoid the case where the project dies because original author is no longer available to lead it.

Forks are good for fixing bugs or implementing new feature but if the changes are not consolidated in a product the effort is mostly wasted and the product does not evolve.

@kilianc Feel free to agree or disagree with my question. I am curious what other forkers think about this and if there is a change of giving rtail a new life.

**Comment by jfcabral** [comment]
Yeah I too think it is a sad to let `rtail` die, I always though it had so much potential! 

I agree with you that the best way to keep these kind of projects from dying is to have an organization with multiple people guiding it!

**Comment by kferrone** [comment]
Oh dang. Yet another great project with no updates and no good alternatives . . .

**Comment by kilianc** [comment]
I am trying to pick this up this weekend!

**Comment by Altiano** [comment]
There are many solution for open source project to get funds

**Comment by IndigoBee** [comment]
> I am trying to pick this up this weekend!

@kilianc Hey Kilian, so did you look into it ?
